app/src/main/java/net/aiscope/gdd_app/repository/SampleRepositorySharedPreference.kt

```diff
@@ -21,7 +23,7 @@ class SampleRepositorySharedPreference @Inject constructor(
     override suspend fun create(): Sample {
         val uuid = uuid.generateUUID()
         val facility = healthFacilityRepository.load()
-        val sample = Sample(uuid, facility.id, facility.microscopist)
+        val sample = Sample(uuid, facility.id, facility.microscopist, createdOn = Calendar.getInstance())
```

Comment:
👍 
In the future, when Android will support them, we could use `java.time` classes. But for now `Calendar` seems to be a good choice.